runtime/tests/js/webapis_tests.js

```diff
@@ -10,6 +10,38 @@ test("atob & btoa", () => {
   assertEquals(atob("c29tZSB0ZXh0"), "some text", `atob("c29tZSB0ZXh0")`);
 });
 
+await test("fetch", async () => {
+  const res = await fetch("https://google.com/");
+  assertEquals(res.status, 200);
+  const text = await res.text();
+  assert(text);
```

Comment:
It would be nice to check that `text` is a string and contains some fragment that's specific enough to give us confidence in response parsing & text decoding, but that's also generic enough to keep the test passing as the web page changes over time.

How about something like this?

```js
assertEquals(typeof text, "string");
assert(text.matches(/<body>/));
```